### PR TITLE
fix: app crashes when editing table visibility

### LINKED ISSUE #7382: [Bug]Production : App crashes on removing Visible data in a table widget
## Description

[What happened]App crashes on removing Visible data in a table widget

### Steps to reproduce the behaviour:

1. Drag and drop the table widget
2. inside the property pane
3. under the column section click on the setting icon
4. Delete Computed Value 
5. now click on the js button for visible 
6. Now delete the true text click outside
7. the application is breaking

Below is the link for more detailes
https://drive.google.com/file/d/1Kh4T20wcGX0gB4b21pTKOI_p9Kfaa4_f/view

### Important Details

- Version: [Cloud ]
- OS: [e.g.MacOSX]
- Browser chrome
- Environment [production,]

### LINKED ISSUE #7448: [Bug] TypeError: e.split is not a function
Sentry Issue: [APPSMITH-14Y](https://sentry.io/organizations/appsmith/issues/2648189277/?referrer=github_integration)

```
TypeError: e.split is not a function
  at f (components/editorComponents/CodeEditor/index.tsx:289:25)
  at componentDidUpdate (components/editorComponents/CodeEditor/index.tsx:281:17)
  at apply (components/editorComponents/CodeEditor/index.tsx:363:10)
  at focus (../../src/helpers.ts:87:17)
  at HTMLDivElement.r (../../src/helpers.ts:87:17)
...
(32 additional frame(s) were not displayed)
```

### PR DESCRIPTION
**Pull request by Tooluloope**
## Description

Fix App crash when editing table visibility

Fixes #7382

Should also fix #7448

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:red_circle: Total coverage has decreased</summary>

    // Code coverage diff between base branch:release and head branch: fix/app-crashing-editing-table-visibility 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :red_circle: | total | 54.88 **(-0.01)** | 36.71 **(0)** | 34.21 **(0)** | 55.42 **(0)**
 :green_circle: | app/client/src/components/editorComponents/CodeEditor/codeEditorUtils.ts | 95.65 **(0)** | 90 **(1.11)** | 71.43 **(0)** | 95 **(0)**
 :green_circle: | app/client/src/components/editorComponents/CodeEditor/index.tsx | 67.79 **(0)** | 40.38 **(0.77)** | 52.5 **(0)** | 68.57 **(0)**
 :red_circle: | app/client/src/components/propertyControls/ComputeTablePropertyControl.tsx | 33.93 **(-0.03)** | 19.05 **(-2)** | 22.22 **(0)** | 33.33 **(0)**
 :red_circle: | app/client/src/utils/autocomplete/TernServer.ts | 50.47 **(-0.24)** | 38.26 **(-0.87)** | 36.21 **(0)** | 55.08 **(-0.27)**</details>

### CODE REVIEW

**Review by jsartisan — APPROVED** [review]

**Comment by YogeshJayaseelan** [comment]
@Tooluloope  - The app still crashes by following the below steps 

1.Drag and Drop table widget 
2.Open property pane and choose any of the column and click on settings icon
3.Remove computed value 
4.Click on visible JS and remove the value 
5.Re-open property pane again 
6.Click on the same column setting icon and observe

https://loom.com/share/db769a2646b1468d8408e960ae32660a

File: app/client/src/components/propertyControls/ComputeTablePropertyControl.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import {
 import { ColumnProperties } from "widgets/TableWidget/component/Constants";
 import { isDynamicValue } from "utils/DynamicBindingUtils";
 import styled from "styled-components";
+import { isString } from "../../utils/helpers";
```
Comment (riodeuno) [inline app/client/src/components/propertyControls/ComputeTablePropertyControl.tsx:16]: Is this the correct import?

Comment (Tooluloope) [reply app/client/src/components/propertyControls/ComputeTablePropertyControl.tsx:16]: Yeah,

**Comment by YogeshJayaseelan** [comment]
@Tooluloope Verified the fix and it is working fine now - App does not crash

**Review by hetunandu — APPROVED** [review]

**Review by jsartisan — APPROVED** [review]

**Comment by jsartisan** [comment]
@techbhavin Also, I'll make few small style updates in the widget.

**Comment by jsartisan** [comment]
@Tooluloope sorry that comment was meant for some other PR. ignore that

**Pull request merged**